### PR TITLE
Feature/dmp 728 mock  offer on payment

### PR DESCRIPTION
**Pull request by dianabasilio**
## What is done

[[DMP-728] show offers on payment](https://deacero.atlassian.net/browse/DMP-728)

## How is done

- Added mock Offers data to payment view.
- Created selectedOffer type, this will be used on the context that we will create later.

## How to test

1. Run the project via `npm start`.
2. Click on any product.
3. Click `Agregar al carrito` button.
4. Go to `carrito` then to payment.
5.  On payment file line 50 you can change the offer type to, Express, Estándar or  Similares depending wich you want to see: 
image

### CODE REVIEW

File: src/config/api/offers.types.ts [hunk 1]
```diff
@@ -19,6 +19,12 @@ export type Offer = {
   total: number;
 };
 
+export type selectedOffer = {
+  title: string;
+  description: string;
+  delivery_options: { delivery_days: string; articles: any }[];
```
Comment (markriverac) [inline src/config/api/offers.types.ts:25]: I think this prop should be an array of Shipments. You can find the info you need in there

File: src/views/Payment/Payment.tsx [hunk 2]
```diff
@@ -177,6 +230,9 @@ const Payment: React.FC = () => {
     </Button>
   );
 
+  console.log("selectedOfferData");
+  console.log(selectedOfferData);
```
Comment (markriverac) [inline src/views/Payment/Payment.tsx:234]: can you remove these

**Review by markriverac — CHANGES_REQUESTED** [review]
Just some small changes. 
And please remove the commits that don't belong here.

File: src/config/api/offers.types.ts (hunk 1 above)
Comment (dianabasilio) [reply src/config/api/offers.types.ts]: I decided to use Offer type, later with the real data I´ll be adding more adjusments to this.

**Review by markriverac — APPROVED** [review]
Yes. It's better this way so the adaptation will be easier. Thank you! <3

**Pull request merged**